.changesets/do-not-report-puma--connectionerror-when-instrumenting-response-bodies.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: patch
+type: change
+---
+
+Do not report `Puma::ConnectionError` when instrumenting response bodies to avoid reporting errors that cannot be fixed by the application.
```

Comment:
This seems misleading -- we're not really filtering for `Puma::ConnectionError`, and in fact other `Puma::ConnectionError` instances should still get through.

```suggestion
Do not report errors caused by `Errno::EPIPE` (broken pipe errors) when instrumenting response bodies, to avoid reporting errors that cannot be fixed by the application.
```